src/main/java/dev/jbang/cli/App.java

```diff
@@ -133,7 +133,7 @@ private static void installScripts(String name, String scriptRef, boolean benati
 	private static void installShellScript(Path file, String scriptRef, boolean benative) throws IOException {
 		List<String> lines = Arrays.asList(
 				"#!/bin/sh",
-				"eval \"exec jbang run" + (benative ? " --native " : " ") + scriptRef + " $*\"");
+				"exec jbang run" + (benative ? " --native " : " ") + scriptRef + " \"$@\"");
```

Comment:
so it looks okey but exec creates a process where eval is in-process - does that change things if we have a long running java app...will ctrl-c and similar still kill the process or can we end up with detached processes?

Reply:
Isn't `exec` used to replace the current process? So shouldn't that handle ctrl+Cs just fine? 🤔

Reply:
I've done some simple testing and ctrl+c seems to work without any problems.